src/Pages/People/index.js

```diff
@@ -0,0 +1,5 @@
+const People = () => {
+  return <p>People</p>;
+};
+
+export default People;
```

Comment:
I'd like to suggest *removing People component* from Navigation folder. It do not directly relate to navigation. In future, we will create *PeopleList* feature instead.